pkg/build/webhook/generic/generic.go

```diff
@@ -43,67 +43,74 @@ func (p *WebHookPlugin) Extract(buildCfg *api.BuildConfig, secret, path string,
 	}
 
 	if buildCfg.Spec.Source.Git == nil {
-		glog.V(4).Infof("No git source defined for BuildConfig %s/%s, but triggering anyway", buildCfg.Namespace, buildCfg.Name)
-		return revision, envvars, true, err
+		warning := webhook.NewWarning("BuildConfig has no git source defined, ignoring and continuing with build")
+		return revision, envvars, true, warning
 	}
 
 	contentType := req.Header.Get("Content-Type")
 	if len(contentType) != 0 {
 		contentType, _, err = mime.ParseMediaType(contentType)
 		if err != nil {
-			return nil, envvars, false, fmt.Errorf("non-parseable Content-Type %s (%s)", contentType, err)
+			return revision, envvars, false, fmt.Errorf("Error parsing Content-Type: %s", err)
```

Comment:
i don't know the reason behind it, but we never uppercase the start of our error messages.  possibly because they end up embedded in other output and look strange.